internal/core/component.rs

```diff
@@ -62,14 +62,13 @@ pub struct ComponentVTable {
     /// And the return value is passed by &mut because ItemWeak has a destructor
     pub get_item_tree: extern "C" fn(core::pin::Pin<VRef<ComponentVTable>>) -> Slice<ItemTreeNode>,
 
-    // FIXME: This does return an invalid ItemWeak now that points to the parent repeater!
-    // FIXME: Get rid of the index and make this always return the "Item" that connects this component
-    //        to its parent-component?
-    /// Return the parent item.
+    /// Return the node this component is a part of in the parent component.
+    ///
     /// The return value is an item weak because it can be null if there is no parent.
     /// And the return value is passed by &mut because ItemWeak has a destructor
-    pub parent_item:
-        extern "C" fn(core::pin::Pin<VRef<ComponentVTable>>, index: usize, result: &mut ItemWeak),
+    /// Note that the returned value will typically point to a repeater node, which is not
+    /// strictly speaking an Item at all!
```

Comment:
Fix the meaning of the sentence to match the intention ;-)

```suggestion
    /// Note that the returned value will typically point to a repeater node, which is
    /// strictly speaking not an Item at all!
```